agent/php_internal_instrument.c

```diff
@@ -796,6 +800,34 @@ static void nr_php_instrument_datastore_operation_call(
   }
 }
 
+/* 
+ * Like nr_php_instrument_datastore_operation_call() but does not
+ * call the underlying internal function
+ */
```

Comment:
Can you include some context as to why we specifically don't want to call the underlying internal function?

Reply:
further details added in [c8c3637](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/c8c3637f08b6375f7c92902dada51b124ba977e6)